docs/explanation/ref-arch-explanation.md

```diff
@@ -0,0 +1,140 @@
+# UP4W architecture
+
+This page describes the different components of UP4W and how they integrate
+together to form the software architecture.
+
+## Background
+
+### What is Ubuntu on WSL?
+
+With the Windows Subsystem for Linux (WSL), Linux distributions can be run on
+Windows with minimal overhead. Ubuntu on WSL is an image that is optimised for
+running Ubuntu on WSL. A user can create an Ubuntu WSL instance on a Windows
+machine and use that instance as a production-ready development environment.
+Ubuntu on WSL also benefits from tight integration with Ubuntu Pro and
+Landscape, which makes it easier to secure Ubuntu WSL instances and manage them
+at scale.
```

Comment:
Good point!

This actually stems from a very recent conversation we had when (re)designing the download page for "Ubuntu WSL". The point was made that expanding Ubuntu WSL into "Ubuntu Windows Subsystem for Linux" isn't as clear as "Ubuntu Desktop" or "Ubuntu Server", especially when "Microsoft WSL" is a legitimate construction. It was a brief exchange, aimed at creating consistency in our documentation and marketing.

Now clearly I am mixing and matching in this explanation! This is partially because repeating "Ubuntu on WSL" gets unwieldy.

I don't think this PR is the place to force through this new convention and I think we should get agreement with engineering/product/marketing so that we can handle it all together.

For now, I'd be happy to revert to Ubuntu WSL in the meantime.